test.py

```diff
@@ -193,7 +195,7 @@ def parse_cmd_line() -> argparse.Namespace:
                              "CLUSTER_POOL_SIZE can be used to achieve the same")
     parser.add_argument('--manual-execution', action='store_true', default=False,
                         help='Let me manually run the test executable at the moment this script would run it')
-    parser.add_argument('--byte-limit', action="store", default=None, type=int,
+    parser.add_argument('--byte-limit', action="store", default=randint(0, 2000), type=int,
```

Comment:
I have absolutely no idea what this means and the help message doesn't help.
Moreover, what the heck is `randint(0,2000)` supposed to do? Is it fine that in some runs the number will be 0? :-(

Reply:
This code is originally from [PR](https://github.com/scylladb/scylladb/pull/22030/files#diff-3665d65394f4f58a56a256ad6dd8621c68118d90fe56a19387e251c19cec2d2e). I'm just moving to another place.

Reply:
Maybe, but if you want to do this change you'll need to understand it enough to answer my question :-(

Reply:
This is a parameter for `toxiproxy` is for setting the limit after which connection will be closed on proxy.

Reply:
If it's a "parameter for toxiproxy", do we really need to set it in the top-level config file? Is setting it to 0 (which will happen in 1/2001 of the test runs) fine?

Reply:
Yes, we need to set it in the top-level to have the ability to pass it through the command line.
If it was previously fine to have 0, why it should not be fine now?

Reply:
I don't know if it was fine or not, I never saw this code before. As I said there is only a 1/2001 chance to get exactly zero. This code seems so random, literally ;-) Why is this even important? What the heck is toxiproxy and what the heck is it's byte limit? Do you understand this code? Can someone who understands this code (maybe @elcallio ?) please come here to review it?

Reply:
> What the heck is toxiproxy and what the heck is it's byte limit?

https://github.com/Shopify/toxiproxy
[bytes](https://github.com/Shopify/toxiproxy?tab=readme-ov-file#limit_data): number of bytes it should transmit before connection is closed

Reply:
Thanks. Now I know what toxiproxy is, but I still have no idea why instead of a specific test controlling its parameters - we have some global parameter. What kind of test can we run using a proxy that breaks every connection after 0 (!) bytes? Or 7 bytes? Why is this parameter even random?

I guess I shouldn't insist here - as you said this is pre-existing code - but there's a difference between strange code deep inside some ldap-specific test that only the test's author understands - and moving this strange code to the top level test configuration.

@elcallio please confirm that this code movement makes sense, and that the random byte-limit setting actually makes sense.

Reply:
Uhm, I have never seen this nor the originating parameter usage before. Afaik, I've had very little to do with LDAP in general, and the python test helpers none at all. 
Personally, I would keep the param as None on this level unless set, and handle None-case in the eventual destination. But that is just me...

Reply:
It's used to test that our ldap client is able to withstand randomly broken connections.